ledger/ledger-on-sql/src/main/resources/com/daml/ledger/on/sql/migrations/h2/V1__create_tables.sql

```diff
@@ -12,13 +12,8 @@ CREATE TABLE ${table.prefix}meta
 CREATE TABLE ${table.prefix}log
```

Comment:
I'm fine with you modifying migrations at this point but you'll need to run `./ledger/ledger-on-sql/hash-migrations.sh` and check in the new hash files.

Reply:
Will run the hashes. Sorry. I only updated the migrations in-place, because you did this just a few days ago. Should I create a new migration?

Reply:
No, it's fine; you'll just have a test failure because of the hashes if you don't run that script.